.github/workflows/pkg-platform.yml

```diff
@@ -0,0 +1,210 @@
+name: pkg for platform
+run-name: pkging ${{ inputs.project }} (${{ inputs.name }})
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: >
+          pretty name for the workflow to make GitHub Actions matrix output
+          more legible
+        required: false
+        type: string
+      os:
+        required: true
+        type: string
+      container:
+        required: false
+        type: string
+      pkg:
+        description: eg. `example.com@1.2.3`
+        required: true
+        type: string
+      dry-run:
+        description: dry runs do not modify bottle storage
+        type: boolean
+        default: false
+      test-os:
+        description: a JSON array of runner-names
+        required: true
+        type: string
+      test-container:
+        description: >
+          A JSON array of docker image names or `[null]`.
+          Indeed! You cannot leave this as `null` or undefined.
+          Sorry, GHA is not flexible enough to efficiently work around this.
+        required: true
+        type: string
```

Comment:
simpler to make it `default: '[null]'`?

Reply:
I think I couldn't easily get this to work because the github actions expressions capabilities are too inadequate.